gateway-luna-helm-sample/headless-helm-example/README.md

```diff
@@ -0,0 +1,21 @@
+## Container GW Headless deployment in Kubernetes without Policy Manager(GW 11.1.2)
+1. Set disklessConfig.enabled to false. Create node.properites file with node.cluster.pass value set. Sample node.properties is for Derby configuration
+   Reference README for APIM Gateway charts GibHub Pages [Diskless Configuration](https://github.com/CAAPIM/apim-charts/blob/stable/charts/gateway/README.md#diskless-configuration) for MySQL database configuration
+2. create Kubernetes Secret
+```
+   kubectl create secret generic gateway-secret --from-file=node.properties=./node.properties
+```
+3. Encrypt LunaPIN by OpenSSL, password is value from node.cluster.pass
+```
+   echo -n '<LunaPIN>' | openssl enc -e -aes-256-cbc -pbkdf2 -iter 600000 -saltlen 16 -pass pass:<gateway_cluster_passphrase> -a;history -d $(history 1)
+```
+4. Append the following system property to config.systemProperties and place encrypted LunaPIN to com.l7tech.encryptedLunaPin in luna_headless_value.yaml
```

Comment:
this section needs a bit formatting, under preview, the formatting is a bit off

<img width="1252" alt="image" src="https://github.com/user-attachments/assets/e2d97755-69eb-44fe-a4d1-1e94151cd3c5">

Reply:
Looks fine when I preview it

<img width="1125" alt="Screenshot 2024-11-12 at 5 29 42 PM" src="https://github.com/user-attachments/assets/7057a38b-81bc-4a8b-ba19-6c81fed52dad">

Reply:
fixed